app/views/play/level/LevelGoalsView.coffee

```diff
@@ -53,7 +53,7 @@ module.exports = class LevelGoalsView extends CocoView
     @previousGoalStatus ?= {}
     @succeeded = e.overallStatus is 'success'
     for goal in e.goals
-      state = e.goalStates[goal.id]
+      state = e.goalStates[goal.id] ? 'incomplete'
```

Comment:
Make sure this is needed after `GoalManager` changes for `headless` option.

Reply:
For new Ozaria levels, this could be useful to always show "incomplete" for new levels with no status.